code/game/area/areas/ruins/_ruins.dm

```diff
@@ -8,6 +8,7 @@
 	dynamic_lighting = DYNAMIC_LIGHTING_FORCED
 	ambientsounds = RUINS
 	sound_environment = SOUND_ENVIRONMENT_STONEROOM
+	valid_territory = FALSE // hey so what if we did not allow things like cult summons to appear on ruins
```

Comment:
earmarked change?

Reply:
the ninja bomb objective functions on valid territory
unfortunately the nearstation ruins are apparently valid territory
double unfortunately cult summons ALSO use valid territory i believe basically asteroids shouldn't be valid territory

Reply:
Makes me wonder if that affects the summon rune on lavaland but wouldnt know